low-level/files/opt/seagate/sspl/bin/sspl_reset.py

```diff
@@ -0,0 +1,174 @@
+import sys
+import errno
+import argparse
+import inspect
+import traceback
+import os
+import shutil
+import subprocess
+# Add the top level directories
+sys.path.insert(0, '/opt/seagate/cortx/sspl/low-level')
+
+from framework.base.sspl_constants import PRODUCT_FAMILY, file_store_config_path, enabled_products
+
+SSPL_CONFIGURED=f"/var/{PRODUCT_FAMILY}/sspl/sspl-configured"
+
+if os.path.exists(file_store_config_path):
+    with open(file_store_config_path, "r") as SSPL_CONF:
+        for line in SSPL_CONF:
+            if 'data_path' in line:
+                SSPL_DATA_DIR=line.split("=")[1]
+            elif 'log_file_path' in line:
+                IEM_FILE_PATH=line.split("=")[1]
+else:
+    SSPL_DATA_DIR=f"/var/{PRODUCT_FAMILY}/sspl/data"
+    IEM_FILE_PATH=f"/var/{PRODUCT_FAMILY}/sspl/data/iem/iem_messages"
+
+class Cmd:
```

Comment:
Cmd class will stay in sspl_setup.py, which will import required classes from here. For testing, you may use code from sspl_setup.py PR

Reply:
Done.